assets/js/components/dashboard/Audit.jsx

```diff
@@ -62,21 +108,31 @@ class Audit extends Component {
 }
 
 const query = gql`
-  query AuditTrailsQuery {
-    auditTrails {
-      id
-      userEmail
-      object
-      action
-      description
-      updatedAt
+  query AuditTrailsQuery ($page: Int, $pageSize: Int) {
+    auditTrails (page: $page, pageSize: $pageSize) {
+      entries {
+        id
+        userEmail
+        object
+        action
+        description
+        updatedAt
+      },
+      totalEntries,
+      totalPages,
+      pageSize,
+      pageNumber
     }
   }
 `
 
 const queryOptions = {
   options: props => ({
-    fetchPolicy: 'network-only'
+    fetchPolicy: 'network-only',
```

Comment:
what's this?

Reply:
makes it so that queries don't get cached. Without this, new actions don't populate unless you refresh the browser. Making it network-only makes it more real-time

Reply:
ok that's fine for now